### PR TITLE
folding-at-home client for kubernetes helm chart

### PR DESCRIPTION
**Pull request by wjimenez5271**

### CODE REVIEW

File: charts/folding-at-home/LICNESE [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Copyright 2020 richstokes et al
```
Comment (PennyScissors) [inline charts/folding-at-home/LICNESE:1]: typo in filename

File: charts/folding-at-home/questions.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  io.cattle.role: cluster # options are cluster/project
+  io.rancher.app_min_version: 7.3.8
+categories:
+questions:
+  - variable: gpu
+    default: false
+    description: "Run using GPUs (if available)"
+    type: boolean
+    required: true
+    label: GPU Enbaled
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:11]: typo

File: charts/folding-at-home/questions.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - variable: replicas
+    default: 2
+    description: "Number of workers to deploy"
+    type: int
+    label: replicas
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:16]: Capitalize

File: charts/folding-at-home/questions.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    group: "Resource Allocations"
+  - variable: requests.cpu
+    default: "100m"
+    description: "Amount of CPU to allocate per pod"
+    type: string
+    label: CPU requests
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:22]: Capitalize requests

File: charts/folding-at-home/questions.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    group: "Resource Allocations"
+  - variable: requests.memory
+    default: "512Mi"
+    description: "Amount of memory to allocate per pod"
+    type: string
+    label: Memory requests
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:28]: Capitalize requests

File: charts/folding-at-home/questions.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    group: "Resource Allocations"
+  - variable: limits.cpu
+    default: "1000m"
+    description: "Maximum amount of CPU to allocate per pod"
+    type: string
+    label: CPU limits
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:34]: Capitalize limits

File: charts/folding-at-home/questions.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    group: "Resource Allocations"
+  - variable: limits.memory
+    default: "512Mi"
+    description: "Maximum amount of memory to allocate per pod"
+    type: string
+    label: Memory limits
```
Comment (PennyScissors) [inline charts/folding-at-home/questions.yaml:40]: Capitalize limits

File: charts/folding-at-home/Chart.yaml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
+icon: file://icon.svg
```
Comment (PennyScissors) [inline charts/folding-at-home/Chart.yaml:22]: Make sure to update this if you decide to change the directory tree structure to include the v0.1.0 subfolder.

Comment (wjimenez5271) [reply charts/folding-at-home/Chart.yaml:22]: good call

**Pull request merged**